hyper/contrib.py

```diff
@@ -132,7 +132,10 @@ def build_response(self, request, resp):
         response = Response()
 
         response.status_code = resp.status
-        response.headers = CaseInsensitiveDict(resp.headers.iter_raw())
+        response.headers = CaseInsensitiveDict((
+            map(lambda h: h.decode("utf-8"), h)
```

Comment:
This is a little overbroad. To match urllib3 we only want to do this on Python 3: on Python 2 these should stay as bytes.